common/rbnf/de.xml

```diff
@@ -16,7 +16,23 @@ For terms of use, see http://www.unicode.org/copyright.html
                 <rbnfrule value="-x">minus →→;</rbnfrule>
                 <rbnfrule value="x.x">=0.0=;</rbnfrule>
                 <rbnfrule value="0">=%spellout-numbering=;</rbnfrule>
-                <rbnfrule value="1100" radix="100">←%spellout-cardinal-masculine←­hundert[­→→];</rbnfrule>
+                <rbnfrule value="1100" radix="100">←←­hundert[­→→];</rbnfrule>
+                <rbnfrule value="2000">=%spellout-numbering=;</rbnfrule>
+                <rbnfrule value="2100" radix="100">←←­hundert[­→→];</rbnfrule>
```

Comment:
I am trying to say these as the German version of "twenty-one hundred fifty three" and it sounds weird to me.
I suggest you just use `=%spellout-numbering=` for everything above 1999.

Reply:
I have lowered the range from 10000 to 2000 as you requested. I guess we can change our minds in the next few decades if the prevailing wording changes.